File: easy school/fees/Fees_Admin.vb
Public Class Fees_Admin

    Private Sub AddNewFeesItemToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles AddNewFeesItemToolStripMenuItem.Click
        Panel1.Controls.Clear()
        Dim residents As New type
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub

    Private Sub CreateOrModifyToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CreateOrModifyToolStripMenuItem.Click
        Panel1.Controls.Clear()
        Dim residents As New fees_rates
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub

    Private Sub BillToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles BillToolStripMenuItem.Click
        Panel1.Controls.Clear()
        Dim residents As New billing
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub

    Private Sub Fees_Admin_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    End Sub

    Private Sub BilllsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles BilllsToolStripMenuItem.Click
        Panel1.Controls.Clear()
        Dim residents As New bill_student
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub

    Private Sub ProjectionsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ProjectionsToolStripMenuItem.Click

    End Sub

    Private Sub YearToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles YearToolStripMenuItem.Click
        Panel1.Controls.Clear()
        Dim residents As New report
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub

    Private Sub ToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles ToolStripMenuItem1.Click
        Panel1.Controls.Clear()
        Dim residents As New bill
        residents.Size = Panel1.Size
        residents.TopLevel = False
        residents.Parent = Panel1
        residents.Show()
    End Sub
End Class

File: easy school.ConvertedToC#/fees/Fees_Admin.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
using System.Threading.Tasks;
namespace easy_school
{
	public partial class Fees_Admin
	{

		private void AddNewFeesItemToolStripMenuItem_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			type residents = new type();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}

		private void CreateOrModifyToolStripMenuItem_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			fees_rates residents = new fees_rates();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}

		private void BillToolStripMenuItem_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			billing residents = new billing();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}


		private void Fees_Admin_Load(object sender, EventArgs e)
		{
		}

		private void BilllsToolStripMenuItem_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			bill_student residents = new bill_student();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}


		private void ProjectionsToolStripMenuItem_Click(object sender, EventArgs e)
		{
		}

		private void YearToolStripMenuItem_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			report residents = new report();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}

		private void ToolStripMenuItem1_Click(object sender, EventArgs e)
		{
			Panel1.Controls.Clear();
			bill residents = new bill();
			residents.Size = Panel1.Size;
			residents.TopLevel = false;
			residents.Parent = Panel1;
			residents.Show();
		}
		public Fees_Admin()
		{
			Load += Fees_Admin_Load;
			InitializeComponent();
		}
	}
}
